Detectors/ITSMFT/common/workflow/src/STFDecoderSpec.cxx

```diff
@@ -124,13 +130,21 @@ void STFDecoder<Mapping>::run(ProcessingContext& pc)
 
   if (mDoDigits) {
     pc.outputs().snapshot(Output{orig, "DIGITS", 0, Lifetime::Timeframe}, digVec);
-    pc.outputs().snapshot(Output{orig, "ITSDigitROF", 0, Lifetime::Timeframe}, digROFVec);
+    if (detID == o2::detectors::DetID::ITS) {
+      pc.outputs().snapshot(Output{orig, "ITSDigitROF", 0, Lifetime::Timeframe}, digROFVec);
+    } else {
+      pc.outputs().snapshot(Output{orig, "MFTDigitROF", 0, Lifetime::Timeframe}, digROFVec);
+    }
```

Comment:
could you instead change all ``ITSDigitROF`` and ``MFTDigitROF`` to ``DigitROF``, the same for ``ClusterROF``? The  DataDescription always comes with DataOrigin, no need for extra naming scheme which make the code more complex